dist/index.html

```diff
@@ -1,18 +1,18 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
 <head>
-    <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Block-Chain Summit</title>
-    <link rel="stylesheet"
-          href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+            rel="stylesheet"
+            href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
     />
     <link rel="stylesheet" href="./css/style.css">
+    <title>PBlock-Chain Expo</title>
 </head>
 <body>
-<header class="header-container">
+<header class="header-wrapper">
```

Comment:
### HOME AND ABOUT PAGE

- Hey, let's improve the design on the navs in the header section to match the design as much as you can. see the images below to do the following:

   - Align the contents correctly 
   - Add a border-bottom to the header to show the end of the header distinguishing from the background.
   - improve the spacing

*main design*
![image](https://github.com/mohashyne/Module1_Capstone/assets/59397159/23263e25-9a8f-46ff-bf40-76c700caf28d)

*your design*
![image](https://github.com/mohashyne/Module1_Capstone/assets/59397159/afd474d6-cfdb-43aa-8738-8c1aea144407)

Reply:
changes Applied and fixed.